File: src/main/java/com/github/k0zka/contentcompress/SubdirsFilter.java
package com.github.k0zka.contentcompress;

import java.io.File;
import java.io.FilenameFilter;

final class SubdirsFilter implements FilenameFilter {

	public boolean accept(final File dir, final String name) {
		return new File(dir, name).isDirectory() && !"WEB-INF".equals(name);
	}

}

File: src/main/kotlin/com/github/k0zka/contentcompress/SubdirsFilter.kt
package com.github.k0zka.contentcompress

import java.io.File
import java.io.FilenameFilter

internal class SubdirsFilter : FilenameFilter {

	override fun accept(dir: File, name: String): Boolean {
		return File(dir, name).isDirectory && "WEB-INF" != name
	}

}